data/jobs.txt

```diff
@@ -1379,6 +1379,80 @@ mission "Bounty Hunting (Small)"
 		payment 150000
 		dialog "The government of <planet> gratefully pays you <payment> for eliminating the <npc>."
 
+mission "Bounty Hunting (Small, boarding, entering)"
```

Comment:
`Boarding, Entering` to match existing mission identifier style.

I also might consider relocating these new missions, and the existing `Bounty Hunting` missions, to the end of the file where they can co-locate with the Bounty Hunting (Marauder) versions.